src/Debug/DataHandlerRegistry.php

```diff
@@ -0,0 +1,123 @@
+<?php
+declare(strict_types = 1);
```

Comment:
Didn't we agree to not put spaces around this? Since that's in the draft PSR.

Reply:
You are the one who added this rule to the PHP CS fixer ;)